### DISCUSSION TITLE
Fooocus, cannot inpaint human

### DISCUSSION THREAD

**Discussion by hschandler** [lllyasviel/Fooocus#3102] [Q&A] [answered]
I can select an area and brush it to add any object for example a dog. But whenever the prompt is "a man", "a girl", it will just give me back the original image (I can see along the process it's generating a person-like outline but disappear at the end...)

**Comment by mashb1t** [comment, marked as answer]
please change the inpaint mode and experiment with the different settings, i can confirm it does indeed work.

**Reply by hschandler** [reply]
Thanks for the confirmation, what does nt stand for? BTW, I can occasionally generate a not-so-good human shape object now with no style check using Juggernaut XL9, but the chance is still very low
